libs/Cargo.lock

```diff
@@ -1296,7 +1296,7 @@ dependencies = [
  "async-trait",
  "base64 0.21.5",
  "bech32",
- "bitcoin 0.29.2",
+ "bitcoin 0.30.2",
```

Comment:
:thinking: I wonder why this updates the bitcoin and lightning-invoice crate as well (as the only crate changed was reqwest) though there don't seem to be any problems with building, so should be alright.

Reply:
It's possible this selected-in or selected-out other dependencies too, based on how they set their own dependency ranges for `reqwest`.